component/card/card.css

```diff
@@ -0,0 +1,144 @@
+
+  .card-hor {
+    display: flex;
+    width: 100%;
+    max-width: 700px;
+    border-radius: 0.5rem;
+    margin-bottom: 2rem;
+    background-color: var(--light-color);
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  }
+  .card-hor-image-ctn {
+    flex: 0.7;
+    padding: 3%;
+  }
+  .card-hor-image-ctn img {
+    border-radius: 1rem;
+  }
+  
+  .card-hor-text-ctn {
+    flex: 1;
+    padding: 1rem;
+    box-sizing: border-box;
+  }
+  .card-hor-image-ctn {
+    max-width: 170px;
+    height: auto;
+  }
+  .card-img-hor {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+  .discount-price {
+    font-size: 17px;
+    font-weight: 600;
+  }
+  .card-hor-btn {
+    margin-top: 6px;
+  }
+  .original-price {
+    text-decoration: line-through;
+    font-size: 14px;
+    color: var(--light-grey);
+    
+  }
+  
+  .vertical-card-container,.dismiss-card-container {
+    max-width: 250px;
+    height: auto;
+    padding: 1rem;
+    box-sizing: border-box;
+    background-color: var(--light-color);
+    border-radius: 0.5rem;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  
+  }
+  
+  .vertical-outer-ctn{
+      position: relative;
+  }
+  .vertical-image-ctn {
+    flex: 1;
+    position: relative;
+  }
+  .vertical-image {
+    width: 100%;
+    height: 100%;
+    border-radius: 0.5rem;
+  }
+  .vertical-card-text-ctn {
+    flex: 0.5;
+  }
+  .vertical-card-content h4 {
+    margin: 0.5rem 0;
+    font-weight: 400;
+  }
+  .badge-trending {
+    position: absolute;
+    /* opacity: 0.5; */
+    top: 0.8rem;
+    left: 0px;
+    width: 130px;
+    clip-path: polygon(0% 0%, 71% 0%, 53% 50%, 69% 97%, 0 97%);
+  }
+  .vertical-card-outer-ctn{
+    display: flex;
+    gap:4rem;
+    flex-wrap:wrap;
+    
+  }
+  .icon-cross-btn{
+    cursor:pointer;
+    border:none;
+    background-color:transparent;
+    font-size: 1.5rem;
+    position: absolute;
+    top:-15px;
+    right:-15px;
+  }
+  .dismiss-card-ctn img,.dismiss-card-ctn{
+    border-radius:0;
+  }
+  .text-overlay-ctn {
+    opacity: 0.5;
+    position: relative;
+  }
+  .out-of-stock {
+      color:black;
+      font-size:1.2rem;
+      font-weight: 1000;
+      top:40%;
+      z-index:2;
+  }
+  .text-overlay{
+       position: absolute;
+       top:50%;
+       left:50%;
+       transform:translate(-50%,-50%);
+       font-weight: 800;
+  }
+  .text-card-ctn{
+      box-sizing:border-box;
+      max-width:300px;
+      padding:1rem;
+      background-color:var(--light-color);
+      border-radius:0.7rem;
+      box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  }
+  .text-card-content-ctn{
+      flex:1;
+  }
+  .text-card-ctn button{
+      flex:0.6;
+      font-size:1rem;
+      font-weight: 700;
+      padding:0.5rem;
+      border-radius:1rem;
+      background-color:var(--light-pink);
+      color:var(--light-color);
+      border:none;
+      cursor: pointer;
+  }
+  
+  
```

Comment:
just a suggestion..you can have all button-related CSS in buttons.css.

Reply:
yeahh making changes over it !!